Files/ViewModels/PreviewPaneViewModel.cs

```diff
@@ -65,6 +66,7 @@ public UIElement PreviewPaneContent
 
         public PreviewPaneViewModel()
         {
+            App.AppSettings.PropertyChanged += AppSettings_PropertyChanged;
```

Comment:
Don't think in makes much of a difference but I added a Dispose method to unsubscribe this event.